spock/plugins/__init__.py

```diff
@@ -1,20 +1,25 @@
 from spock.plugins.core import auth, event, net, ticker, timer
-from spock.plugins.helpers import clientinfo, entities, inventory, keepalive, \
-    move, physics, respawn, start, world
+from spock.plugins.helpers import clientinfo, entities, interact, inventory,\
+    keepalive, move, physics, respawn, start, world
 
-DefaultPlugins = [
+core_plugins = [
+    ('auth', auth.AuthPlugin),
     ('event', event.EventPlugin),
     ('net', net.NetPlugin),
-    ('auth', auth.AuthPlugin),
     ('ticker', ticker.TickerPlugin),
     ('timers', timer.TimerPlugin),
```

Comment:
Problem is that every plugin requires it with "Timers". This would have to be changed everywhere.

But I agree, keeping it consistent would be better.